Parsers/ASimAuditEvent/Parsers/vimAuditEventMicrosoftWindowsEvents.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
couldn't find data. Did we do data and filtering testing for this parser using ASC?

Reply:
yes